.goreleaser.yml

```diff
@@ -158,14 +155,6 @@ nfpms:
           mode: 0755
           owner: observiq-otel-collector
           group: observiq-otel-collector
-      # TODO: (dakota) evaluate this file once supervisor has support for configuring agent logging
-      - src: release_deps/logging.yaml
-        dst: /opt/observiq-otel-collector/logging.yaml
```

Comment:
Will Windows / Linux agents log to a file? If so, what path?

Reply:
The path is `<agent_home>/supervisor_storage/agent.log`. This is non-configurable by the supervisor - at least for now, unsure if this will change at all. The post install script messages about where to find agent logs were updated with this.

Reply:
Cool, would be cool if it could be configured to go to the current path `/opt/observiq-otel-collector/log/collector.log` but its not a big deal.

Mostly curious.